snap/snapcraft.yaml

```diff
@@ -4,8 +4,114 @@ summary: OpenStack Exporter for Prometheus
 license: MIT
 contact: solutions-engineering@lists.canonical.com
 description: |
-  The OpenStack exporter, exports Prometheus metrics from a running OpenStack
-  cloud for consumption by prometheus.
+  This snap uses openstack-exporter to provide OpenStack cloud metrics via Prometheus.
+  Grafana dashboards can then be used to visualize the exported metrics, see for example:\
+  https://grafana.com/grafana/dashboards/9701-openstack-dashboard/
+
+  **How-To**
+  ---
+
+  **How to install the snap:**
+
+     sudo snap install charmed-openstack-exporter
+
+     # To allow the snap to read from the /etc/openstack/, you may need to manually add the connection:
+     sudo snap connect charmed-openstack-exporter:etc-openstack
+
+  **How to configure the service:**
+
+  The openstack-exporter service is disabled by default and needs to be configured before the startup.\
+  For example:
```

Comment:
Are backslashes necessary? Does snap store markdown auto reflow paragraphs?

Reply:
This way I can force a new line, otherwise it will be one line

Reply:
Looks better on the store :)

Reply:
I prefer that way. Otherwise, each sentence will be separated by two new lines (in the code). Example:

```
 The openstack-exporter service is disabled by default and needs to be configured before the startup.\
  For example:
 ```
 
 ```
  The openstack-exporter service is disabled by default and needs to be configured before the startup.
  
  For example:
  ```
  
  It will give the same results, but with `\`, I can group them, just a preference